test/lib/code_corps/github/adapters/github_repo_test.exs

```diff
@@ -9,7 +9,7 @@ defmodule CodeCorps.GitHub.Adapters.GithubRepoTest do
 
   describe "from_api/1" do
     test "maps api payload correctly" do
-      %{"repositories" => [repo]} = load_fixture("user_repositories")
+      %{"repositories" => [repo]} = load_event_fixture("user_repositories")
```

Comment:
You made a rename to `load_event_fixture`, but in this specific case, you're actually loading an api response fixture. I'm ok with naming, but then we'd need a `load_response_fixture` as well and probably split the two sets of fixtures into separate folders under `fixtures/github/`